### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.1.4

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+### Features and enhancements
+
+- **Explore:** Ensure logs volume bar colors match legend colors. [#39072](https://github.com/grafana/grafana/pull/39072), [@ifrost](https://github.com/ifrost)
+- **LDAP:** Search all DNs for users. [#38891](https://github.com/grafana/grafana/pull/38891), [@sakjur](https://github.com/sakjur)
+
+### Bug fixes
+
+- **Alerting:** Fix notification channel migration. [#38983](https://github.com/grafana/grafana/pull/38983), [@papagian](https://github.com/papagian)
+- **Annotations:** Fixes blank panels for queries with unknown data sources. [#39017](https://github.com/grafana/grafana/pull/39017), [@hugohaggmark](https://github.com/hugohaggmark)
```
Comment (joshhunt) [inline CHANGELOG.md:14]: Fixes -> Fix

@hugohaggmark .

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+- **BarChart:** Fix stale values and x axis labels. [#39188](https://github.com/grafana/grafana/pull/39188), [@leeoniya](https://github.com/leeoniya)
+- **Graph:** Make old graph panel thresholds work even if ngalert is enabled. [#38918](https://github.com/grafana/grafana/pull/38918), [@domasx2](https://github.com/domasx2)
+- **InfluxDB:** Fix regex to identify `/` as separator. [#39185](https://github.com/grafana/grafana/pull/39185), [@dsotirakis](https://github.com/dsotirakis)
+- **LibraryPanels:** Fixes update issues related to library panels in rows. [#38963](https://github.com/grafana/grafana/pull/38963), [@hugohaggmark](https://github.com/hugohaggmark)
+- **Variables:** Fix not updating inside a Panel when the preceding Row uses "Repeat For". [#38935](https://github.com/grafana/grafana/pull/38935), [@axelavargas](https://github.com/axelavargas)
```
Comment (joshhunt) [inline CHANGELOG.md:19]: Fix not -> Fix variables not

@axelavargas

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+- **LDAP:** Search all DNs for users. [#38891](https://github.com/grafana/grafana/pull/38891), [@sakjur](https://github.com/sakjur)
+
+### Bug fixes
+
+- **Alerting:** Fix notification channel migration. [#38983](https://github.com/grafana/grafana/pull/38983), [@papagian](https://github.com/papagian)
+- **Annotations:** Fixes blank panels for queries with unknown data sources. [#39017](https://github.com/grafana/grafana/pull/39017), [@hugohaggmark](https://github.com/hugohaggmark)
+- **BarChart:** Fix stale values and x axis labels. [#39188](https://github.com/grafana/grafana/pull/39188), [@leeoniya](https://github.com/leeoniya)
+- **Graph:** Make old graph panel thresholds work even if ngalert is enabled. [#38918](https://github.com/grafana/grafana/pull/38918), [@domasx2](https://github.com/domasx2)
+- **InfluxDB:** Fix regex to identify `/` as separator. [#39185](https://github.com/grafana/grafana/pull/39185), [@dsotirakis](https://github.com/dsotirakis)
+- **LibraryPanels:** Fixes update issues related to library panels in rows. [#38963](https://github.com/grafana/grafana/pull/38963), [@hugohaggmark](https://github.com/hugohaggmark)
```
Comment (joshhunt) [inline CHANGELOG.md:18]: Fixes -> Fix

@hugohaggmark .

**Review by joshhunt — APPROVED** [review]
I'm not words good, but spotted some inconsistencies with the changelog/pr titles.

Not sure if these are a high priority to fix? Could fix in original PRs and regenerate?

File: CHANGELOG.md (hunk 1 above)
Comment (sunker) [reply CHANGELOG.md:14]: I'll fix :)

File: CHANGELOG.md (hunk 3 above)
Comment (sunker) [reply CHANGELOG.md:18]: I'll fix :)

File: CHANGELOG.md (hunk 2 above)
Comment (sunker) [reply CHANGELOG.md:19]: I'll fix :)

Comment (axelavargas) [reply CHANGELOG.md]: Thank you @joshhunt I updated the PR, @sunker I for fixes we need to update the PRS and then re-run the [Update Changelog job github action](https://github.com/grafana/grafana/actions/workflows/update-changelog.yml) right?

Comment (sunker) [reply CHANGELOG.md]: I fixed it in this PR. Should be enough?

Comment (joshhunt) [reply CHANGELOG.md]: I think the normal approach is to edit the actual PR title, and rerun the action to generate the changelog, that way the edits aren't lost in the next release?

Comment (sunker) [reply CHANGELOG.md]: Ok I'll do it

**Comment by sunker** [comment]
regenerating the changelog

**Pull request closed without merging**